opba-protocols/sandboxes/xs2a-sandbox/src/test/java/de/adorsys/opba/protocol/xs2a/testsandbox/BasicTest.java

```diff
@@ -84,4 +84,21 @@ void startTheSandboxInDocker() {
         // Loop forever.
         await().forever().until(() -> false);
     }
+
+    /**
+     * Not really a test, but just launches entire sandbox for you.
```

Comment:
Please fix JavaDoc comment, it is not starting sandbox, it starts WireMock that imitates Sandbox

Reply:
JavaDoc updated